qcfractal/interface/models/rest_models.py

```diff
@@ -1122,3 +1140,115 @@ class QueueManagerPUTResponse(ProtoModel):
 
 
 register_model("queue_manager", "PUT", QueueManagerPUTBody, QueueManagerPUTResponse)
+
+
+## advanced procedures queries
+
+class OptimizationBestResultsBody(ProtoModel):
+    class Data(ProtoModel):
+        opt_ids: QueryObjectId = Schema(
+            None,
+            description="List of optimization procedure Ids to fetch their BEST results from the database."
```

Comment:
I think we need to call this `Last` or `Final` results to more accurately reflect the operation.